README.md

```diff
@@ -109,7 +109,30 @@ MyLogger.send :include, ActiveRecord::SessionStore::Extension::LoggerSilencer
 This silencer is being used to silence the logger and not leaking private
 information into the log, and it is required for security reason.
 
-## Contributing to Active Record Session Store
+CVE-2015-9284 mitigation
```

Comment:
@thorsteneckel was this just a mistake or does CVE-2015-9284 have any connection to CVE-2019-25025 ?